tests/manual/ble/sysbuild.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+SB_CONFIG_PARTITION_MANAGER=n
```

Comment:
this seems to conflict with the above file `CONFIG_PARTITION_MANAGER_ENABLED=y`?

Reply:
This is a test application, partition manager is not enabled there